apps/synthmark.cpp

```diff
@@ -62,6 +62,7 @@ void usage(const char *name) {
            kDefaultSeconds);
     printf("    -b{burstSize} frames read by virtual hardware at one time, default = %d\n",
            kDefaultFramesPerBurst);
+    printf("    -B{burstNum} initial number of bursts, default = %d\n", 1);
```

Comment:
Was this initial bursts intended to speed up the LatencyMark test?
I am working on an optimized search algorithm so this may not be needed.

Reply:
Yes, instead of starting from a minimum value and then increase, I added this field to directly jump to the minimum number of bursts required (when known).
An optimized search algorithm is absolutely useful, but having the power of setting a known value is also a feature that I found useful during my experiments.
Maybe we could add that if -B is followed by a negative number, then it uses the optimized search algorithm. What do you think?